experiment_prototype/experiment_prototype.py

```diff
@@ -262,11 +264,12 @@ def __init__(self, cpid):
         self.__new_slice_id = 0
 
         # Centre frequencies can be specified in your experiment class using the setter. TODO: make modifiable (with warning that it takes time. Get time estimate for this.
-        self.__txctrfreq = 13000  # in kHz.
-        self.__rxctrfreq = 13000  # in kHz.
+        self.txctrfreq = 12000  # in kHz.
+        self.rxctrfreq = 12000  # in kHz.
 
+        print(self.txctrfreq)
         # Load the config, hardware, and restricted frequency data
-        self.__options = ExperimentOptions()
```

Comment:
I think you may have accidentally removed this line?

Reply:
It was moved to line 257 so we could use the master clock rate in the calculation for freqs.